pkg/github/github.go

```diff
@@ -63,6 +65,7 @@ const compareURLTemplate = "https://api.github.com/repos/%s/compare/%s...%s"
 
 var providerToEndpointPrefix = map[string]string{
 	"github": "https://raw.githubusercontent.com",
+	"github_api": "https://api.github.com",
```

Comment:
This is overloading "providers of repo content" with a mismatching semantic meaning of the new key/val. `github_api` should idealy be a part of `"github"` value in the map somehow.

Reply:
do you have any specific suggestions of how to do this? I also considered just using something like `branchesURLTemplate` - we use that for a few of the API endpoints, and that might make more sense than including it in this map

Reply:
decided to go the `URLTemplate` path, so this is no longer relevant.